lollipop/types.py

```diff
@@ -421,19 +420,19 @@ def load(self, data, *args, **kwargs):
 
         return super(Tuple, self).load(result, *args, **kwargs)
 
-    def dump(self, data, *args, **kwargs):
```

Comment:
This content does not correspond to commit message. Please extract it as a separate commit